.github/pull_request_template.md

```diff
@@ -8,6 +8,7 @@ and then fulfill the remaining checklist items before merging.
 Closes #
 
 # adopt a passed proposal
+
```

Comment:
is the prettier style set up to stick?

or was this just a 1-time?

Reply:
one time, it's not linted